util/testutil/util.go

```diff
@@ -0,0 +1,31 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package testutil
+
+import (
+	"math/rand"
+	"testing"
+)
+
+// RandomString returns a random string of the specified number of bytes
+func RandomString(t *testing.T, n int) string {
+	b := make([]byte, n)
+	if _, err := rand.Read(b); err != nil {
+		t.Fatalf("failed rand.Read: %v", err)
+	}
+	return string(b)
```

Comment:
This can be non-ascii

Reply:
Thank you for the review. I think it's fine that they're non-ascii. Fixed the function name to `RandomBytes`.